### ISSUE TITLE
Author Avatar Instead of Commit Dot?

### ISSUE THREAD

**Issue by fadi-george** [mhutchie/vscode-git-graph#388]
**Describe the feature that you'd like**
Hello, I was wondering if there is a setting that can show the commit's author avatar instead of the dot. If not, it would be a really good feature to have.

**Additional context (optional)**
GitKraken is a tool that does this.

<img width="332" alt="Screen Shot 2020-09-29 at 11 15 26 PM" src="https://user-images.githubusercontent.com/6709400/94649731-20a03c00-02aa-11eb-87a9-00547d6713e4.png">

**Comment by mhutchie** [comment]
Hi @fadi-george, Thanks for raising this feature request!

There are several reasons why I decided that it was best to put the commit's author avatar in the Author column, instead of in the commit dots. All of these reasons were heavily guided and based upon substantial user feedback I've received, so I'm fairly confident the decisions I've made were the best trade-offs for the majority of current users. Additionally, implementing this functionality would require a surprisingly significant amount of changes to the UI of Git Graph, so having both options (controlled via an extension setting) is unfortunately infeasible - I've got to decide on one approach or the other.

GitKraken is the only tool I'm aware of that decided to put the commit's author avatar in the commit dot. I definitely understand that if you're used to using GitKraken, you would prefer commit avatar's in Git Graph to look "just like GitKraken", but that doesn't necessarily mean that it is universally the best solution - as every Git UI evolves based on considerable user feedback to best meet the needs of it's users.

I'll close this feature request for now, and add it to the backlog. If I get more user feedback in the future that indicates that a significant amount of users would prefer the commit's author avatar in the commit dots, I'll happily re-open and implement this functionality. I'm always extremely keen to adapt and evolve Git Graph to whatever the user community is wanting, it's just that I must also be aware of potential trade-offs to ensure every new feature / change is in the best interest of the majority of users.
